portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2281,7 +2281,11 @@ definition {
 			category = "Content &amp; Data",
 			portlet = "Forms");
 
-		LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		Click.clickNoMouseOverNoWaitForVisible(locator1 = "Icon#CURRENT_DISPLAY_STYLE");
+
+		Click.clickNoMouseOverNoWaitForVisible(
+			key_menuItem = StringUtil.upperCaseFirstLetter("list"),
+			locator1 = "MenuItem#ANY_MENU_ITEM");
```

Comment:
@leticiachiu, from what I could understand from this change, your proposal is perform the clicks without waiting for the elements load on the screen, am I right?

If so, won't it go in the opposite direction of what we've discussed yesterday? Can you please clarify?

Thanks

Reply:
Ooh, I misunderstood this macro.. Sorry :dizzy: